types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Minimum TypeScript Version: 3.0
+import {Node} from 'unist'
+
+declare namespace mdastToString {}
```

Comment:
This one is actually quite interesting. This line is supposedly redundant, but it does change how TypeScript behaves.

Typically namespaces are used this way to export named fields, for the unified packages these are often function options. A side effect is this also makes TypeScript shut up about this import:

```ts
import * as mdastToString from 'mdast-util-to-string';
```

Personally I like this, because I don’t see why a namespace declaring only interfaces should change how a function can be imported. However, DefinitelyTyped maintainers tend to disagree.

If we do decide to explicitly support this, I think tests should be changed to support this.

```diff
- import namespace = require('package')
+ import * as namespace from 'package'
```

Reply:
I agree, added `*` import test in b6c3471

Reply:
huh, github may be having a hiccup, I can see the commit on the branch this PR is based on https://github.com/ChristianMurphy/mdast-util-to-string/commit/b6c347111b00d3add981c24c87989999afdb5d9a
but it's not showing in the PR :thinking:

Reply:
51045c0 has the changes